cmd/main.go

```diff
@@ -768,6 +787,13 @@ func deployModule(ctx *Context, system *config.System, module string) error {
 		return err
 	}
 
+	// Some repos apply examples (e.g. nnf-sos' container/storage profiles) in an additional step in
+	// deploy.sh, so account for an additional overlay to use in that case.
+	overlayExample, err := getExampleOverlay(ctx, system, module)
```

Comment:
This function will come up empty since there are no `examples-*` overlays listed in `systems.yaml` for KIND, so it should default to deploying the regular `config/examples`. 

However, this make me discover a bug when deploying to KIND, which I just pushed out a fix.